src/basic/conf-files.c

```diff
@@ -261,16 +261,12 @@ int conf_files_list_strv(char ***strv, const char *suffix, const char *root, uns
         return conf_files_list_strv_internal(strv, suffix, root, flags, copy);
 }
 
-int conf_files_list(char ***strv, const char *suffix, const char *root, unsigned flags, const char *dir, ...) {
+int conf_files_list(char ***strv, const char *suffix, const char *root, unsigned flags, const char *dir) {
```

Comment:
hmm, i figure we could also get rid of this whole function, and just expect people to use STRV_MAKE() around the last argument if there's one argument only.

Reply:
Dunno, there's a few users of this function and using STRV_MAKE() would make each one a bit more verbose. Since this is already implemented, let's keep it.